p4-16/psa/PSA.mdk

```diff
@@ -444,12 +444,11 @@ the responsibility of the programmer to parse the cloned packet
 such that it correctly extracts the attached metadata.
 The attached metadata may
 be of type `header`, `header stack`, `header union` or `struct` of the
-above types. Invoking the `emit` method multiple times will attach all
-specified metadata to the same cloned packet. The PSA architecture
-instantiates the `clone` extern in the ingress and egress deparser. A
-P4 program can use it in ingress and egress deparser only. It is an
-error to instantiate the `clone` extern in the P4 control or parser
-block.
+above types. Invoking `emit` method multiple times on the same cloned
+packet is not allowed. The PSA architecture instantiates the `clone`
+extern in the ingress and egress deparser. A P4 program can use it in
+ingress and egress deparser only. It is an error to instantiate the
+`clone` extern in the P4 control or parser block.
```

Comment:
The instantiation is still confusing: you say that the PSA instantiates the clone extern -- is that implicit or does the user have to instantiate explicitly? If it is implicit, then there is no need for the later part. If it is explicit, then we should say that PSA only allows their instantiation in the ingress/egress deparser and instantiating anywhere else is an error.